BOUNTY.md

```diff
@@ -0,0 +1,115 @@
+# GraphQL.NET Bounty Program
+
+The GraphQL.NET team has worked to create a program where community developers can work on specific new features
+of GraphQL.NET for compensation. These features are determined by GraphQL.NET maintainers to have the
+potential for high-impact improvement to the product. Design proposals that would fulfill a bounty may
+be made either by individuals or teams of developers, but the final payout will be made to only one stakeholder.
+
+
+## Bounty Process Flowchart
+
+1. The GraphQL.NET team publishes a Request For Proposals (RFP) describing a feature that the team has decided
+   should be prioritized, or a bug that is in need of being researched and fixed.
+    * Initial Bounty Status set to **Accepting Proposals**
```

Comment:
Do you mean that all these statuses need to be formalized as labels?

Reply:
I was going to create a few badges, yes, that correspond to these statuses.